src/test/java/ludogorie_soft/reservations_platform_api/mapper/UserMapperTest.java

```diff
@@ -0,0 +1,76 @@
+package ludogorie_soft.reservations_platform_api.mapper;
+
+import ludogorie_soft.reservations_platform_api.dto.RegisterDto;
+import ludogorie_soft.reservations_platform_api.entity.User;
+import ludogorie_soft.reservations_platform_api.helper.UserTestHelper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.modelmapper.ModelMapper;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class UserMapperTest {
+
+    private static final String TEST_NAME = "Test Name";
+    private static final String TEST_EMAIL = "test@email.com";
+
+    @Mock
+    private PasswordEncoder passwordEncoder;
+
+    @Mock
+    private ModelMapper modelMapper;
+
+    @InjectMocks
+    private UserMapper userMapper;
+
+    private RegisterDto registerDto;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
I prefer using `@ExtendWith(MockitoExtension.class) `annotation on the class instead of `MockitoAnnotations.openMocks(this);`

Reply:
Done!